assets/css/style.css

```diff
@@ -185,7 +185,7 @@ body {
 
 /* === 3.1 HOME BODY === */
 
-body.home {
+/*body.home {
```

Comment:
Makes sense. I did have to change something in order to get the new font to work for the subhead and I wonder if removing this altered the specificity-ness of the override coming from custom.css. We might have to keep an eye on styling to make sure this change wasn't what caused the issue (I was able to change the font without the extra change in one commit, but not a different commit. Wasn't obvious why it wouldn't work but now I'm wondering.... 😝 )